service-api/src/main/java/greencity/dto/achievement/AchievementPostDto.java

```diff
@@ -0,0 +1,19 @@
+package greencity.dto.achievement;
+
+import greencity.dto.achievementcategory.AchievementCategoryDto;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class AchievementPostDto {
```

Comment:
AchievementPostDto  - looks a lot of duplication from AchievementManagementDto. Think of inheritance to composition

Reply:
Done